config/environments/staging.rb

```diff
@@ -28,9 +28,8 @@
   # Apache or NGINX already handles this.
   config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?
 
-  # Compress JavaScripts and CSS.
-  config.assets.js_compressor = Uglifier.new(harmony: true)
-  # config.assets.css_compressor = :sass
+  # Don't add an asset compressor here because we already minimize with webpack.
```

Comment:
Out of curiosity, is there a way to share rails environments so we don't need to copy code like this?

Reply:
looks like yes 👍 https://stackoverflow.com/questions/14947833/remove-duplication-between-staging-rb-and-production-rb Would be a good clean-up task.